python/tvm/contrib/target/android_nnapi/relayir_to_nnapi_converter/_export_object/operand.py

```diff
@@ -0,0 +1,151 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Android NNAPI Operand-related helper methods on ExportObject
+"""
+
+
+class Operand:
+    """Android NNAPI Operand-related helper methods on ExportObject"""
+
+    def __init__(self, export_obj):
+        self._export_obj = export_obj
+
+    def get_dtype(self, idx):
+        """Get operand dtype
+
+        Parameters
+        ----------
+        idx: int
+            operand to be queried
+
+        Returns
+        -------
+        dtype: str
+            dtype of the queried operand
+
+        """
+        return self._export_obj["types"][self._export_obj["operands"][idx]["type"]]["type"]
+
+    def get_shape(self, idx):
+        """Get operand shape
+
+        Parameters
+        ----------
+        idx: int
+            operand to be queried
+
+        Returns
+        -------
+        shape: tuple of int or None
+            shape of the queried operand. None if operand has no shape.
+
+        """
+        return self._export_obj["types"][self._export_obj["operands"][idx]["type"]].get(
+            "shape", None
+        )
+
+    def get_rank(self, idx):
+        """Get operand rank
+
+        Parameters
+        ----------
+        idx: int
+            operand to be queried
+
+        Returns
+        -------
+        rank: int
+            rank of the queried operand
+
+        """
+        shape = self.get_shape(idx)
+        if shape is None:
+            return 0
+        return len(shape)
+
+    def get_value(self, idx):  # pylint: disable=inconsistent-return-statements
+        """Get operand value
+
+        Parameters
+        ----------
+        idx: int
+            operand to be queried
+
+        Returns
+        -------
+        value:
+            value of the queried operand. None if there's no value.
+
+        """
+        value_dict = self._export_obj["operands"][idx].get("value", None)
+        if value_dict is None:
+            return None
+
+        if value_dict["type"] == "constant_idx":
+            return self._export_obj["constants"][value_dict["value"]]["value"]
+        if value_dict["type"] == "memory_ptr":
+            return value_dict["value"]
+        assert False, "Unreachable"
```

Comment:
```suggestion
        assert value_dict["type"] == "memory_ptr":
        return value_dict["value"]
```
So that you don't need to disable inconsistent-return-statements.